src/main/java/trackitnus/logic/parser/ParserUtil.java

```diff
@@ -192,11 +212,13 @@ public static double parseWeightage(String weightage) throws ParseException {
      *
      * @throws ParseException if the given {@code remark} is invalid.
      */
-    public static String parseRemark(String remark) throws ParseException {
-        requireNonNull(remark);
-        String trimmedRemark = remark.trim();
-        if (!Address.isValidAddress(trimmedRemark)) {
-            throw new ParseException(Task.REMARK_MESSAGE_CONSTRAINTS);
+    public static String parseRemark(Optional<String> remark) {
+        if (remark.isEmpty()) {
+            return null;
+        }
+        String trimmedRemark = remark.get().trim();
+        if (!Task.isValidString(trimmedRemark)) {
+            return null;
```

Comment:
I think the remark doesn't need any kind of validity checking right?

Reply:
Same as above. I want to distinguish between storing remark as null vs storing it as an empty string. Unless maybe it is better to just store all null as empty string?

Reply:
yeah store it as an empty string is better